lib/bolt/cli.rb

```diff
@@ -449,9 +449,8 @@ def execute(options)
         when 'init'
           code = initialize_project
         when 'migrate'
-          inv = config.inventoryfile
-          path = config.project.path
-          code = Bolt::ProjectMigrate.new(path, outputter, inv).migrate_project
+          ok = Bolt::ProjectMigrate.new(config, outputter).migrate_project
+          code = ok ? 0 : 1
```

Comment:
I think since this is called directly from the CLI class it's appropriate for the migrate_project method to just return 0 or 1. It's only methods that aren't directly returned by the CLI where that doesn't make sense.